main.go

```diff
@@ -258,20 +271,20 @@ func (x *x32) listen(wait time.Duration) (any, error) {
 	case 'f':
 		return msg.Args[0].Float32(), nil
 	}
-	return nil, fmt.Errorf("cannot determine data type:")
+	return nil, errors.New("cannot determine data type")
 }
 
 func (x *x32) sendAndListen(message string, wait time.Duration) (any, error) {
 	err := x.SendString(message)
 	if err != nil {
 		return "", err
 	}
-	out, err := x.listen(wait)
+	out, err := x.listen(context.Background(), wait)
 	return out, err
 }
 
 func (x *x32) getChFader(ch int) (float32, error) {
-	chS := fmt.Sprintf("%d", ch)
+	chS := fmt.Sprintf("%02d", ch)
 	if ch < 10 {
```

Comment:
This conditional needs to be removed after adjusting the formatting in fmt.Sprintf to pad zeros to a width of 2